File: CTSWebApp/ClientApp/model/CalendarWeek.ts
export class CalendarWeek {

    constructor(id:number, calendarYearId :number, weekNo :number, description :string, termNo : number) {
        this.Id = id;
        this.CalendarYearId = calendarYearId;
        this.WeekNo = weekNo;
        this.Description = description;
        this.TermNo = termNo;
    }

    Id: number;
    CalendarYearId: number;
    WeekNo: number;
    Description: string;
    //WeekDate: any;
    TermNo: number;
}

File: CTSWebApp/Data/Entities/CalendarWeek.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace CTSWebApp.Data.Entities
{
    [Table("CalendarWeek")]
    public class CalendarWeek
    {
        [Key]
        public int ID { get; set; }
        public int CalendarYearID { get; set; }
        public int WeekNo { get; set; }
        public string Description { get; set; }
        public DateTime WeekDate { get; set; }
        public int TermNo { get; set; }
        public string Active { get; set; }
    }
}
